packages/cli/test/unit/controllers/me.controller.test.ts

```diff
@@ -223,7 +223,7 @@ describe('MeController', () => {
 	describe('API Key methods', () => {
```

Comment:
Do not think this test suite is needed. We are testing this endpoints in `me.api.test.ts`